flake-modules/dev/list-plugins.py

```diff
@@ -31,10 +59,71 @@ class State(Enum):
     OLD = "❌"
 
 
-KNOWN_PATHS: dict[str, tuple[State, Kind, bool]] = {
-    # "plugins/utils/mkdnflow.nix": (True, Kind.NEOVIM),
+KNOWN_PATHS: dict[
+    str,
+    tuple[
+        State,  # If the implem is "legacy" or up to date
+        Kind,  # Vim / Neovim / misc
+        bool,  # Has deprecation warnings
+    ],
+] = {
+    "plugins/by-name/chadtree/default.nix": (State.OLD, Kind.NEOVIM, False),
+    "plugins/by-name/commentary/default.nix": (State.OLD, Kind.VIM, False),
+    "plugins/by-name/conjure/default.nix": (
+        State.OLD,
+        Kind.VIM,  # configured with weird globals `vim.g["conjure#mapping#doc_word"] = "gk"`
+        False,
+    ),
```

Comment:
Doesn't having this hard-coded defeat the purpose of this test? Can you try and explain why we're doing things this way?

Reply:
Well, for me, having a plugin listed as "unknown" is not helping at all.
So we have two choices:
1. improve the matching logic to automatically identify more plugins
2. Add exceptions

But anyway, I pushed a commit that completely get rid of `kind.UNKNOWN`.